src/runtimes/burrego.rs

```diff
@@ -160,9 +160,17 @@ impl<'a> Runtime<'a> {
             }
             Err(err) => {
                 error!(
-                    error = err.to_string().as_str(),
+                    error = ?err,
```

Comment:
I don't understand this casting. Does this mean [that err is optional](https://doc.rust-lang.org/reference/macros-by-example.html#repetitions)?

Reply:
This is not a casting. This is the special syntax used by the `tracing` macros. The question mark means: use the display output of the object (like doing `{:?}` formatting)

Reply:
You can see more details [here](https://docs.rs/tracing/0.1.37/tracing/#recording-fields)

Reply:
doh, thanks!